pkg/proto/modelzooserver.proto

```diff
@@ -58,6 +62,7 @@ message ListModelDefResponse {
     int64 size = 5;
 }
```

Comment:
Shall we add a `ModelDefResponse` to The `ListModelDefResponse`? Like:
``` go
message ListModelDefResponse {
    message ModelDefResponse {
        string class_names = 1;
        string image_urls = 2;
        string tags = 3;
        string arg_descs = 4;
    }
    repeat ModelDefResponse  model_defs = 1;
}
```
So that we don't have to tell users we ensure `len(class_names) = len(image_urls) = ... = size`

Reply:
Yes, that's indeed better.